packages/visx-brush/src/BaseBrush.tsx

```diff
@@ -442,8 +632,13 @@ export default class BaseBrush extends React.Component<BaseBrushProps, BaseBrush
                     stageWidth={stageWidth}
                     stageHeight={stageHeight}
                     updateBrush={this.updateBrush}
-                    brush={this.state}
+                    brush={{ ...this.state }}
```

Comment:
why does this need to be copied now?

Reply:
I'll roll back it. It was added during  searching for some bug in this PR